.gitignore

```diff
@@ -58,3 +58,5 @@ scripts/curl.fish
 curl_fuzzer
 curl_fuzzer_seed_corpus.zip
 libstandaloneengine.a
+cmake-build-debug
+.idea
```

Comment:
What's `.idea` and why should our file ignore that?

Reply:
as a commit message says ".gitignore cmake-build-debug created by CLion"